share/revng/test/tests/analysis/ImportFromCAnalysis/rft-with-an-unannotated-argument/expected-error.txt

```diff
@@ -0,0 +1,6 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Arguments must have either a register or a stack annotation.
```

Comment:
I'd like the syntax of the annotations to be spelled out in the error messages. To help the user conform.

This applies to all errors about annotations in general, not just here.

Reply:
Done (where reasonable).